Functions/Program.cs

```diff
@@ -0,0 +1,12 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+var host = new HostBuilder()
+    .ConfigureFunctionsWorkerDefaults()
+    .ConfigureServices(builder =>
+    {
+        builder.AddHttpClient("PollEmailProcessing");
```

Comment:
We don't need to specify the `HttpClient` name here as we didn't create a named `HttpClient`.

Reply:
Good call, updated